src/DependencyInjection/whatwedoMonitorExtension.php

```diff
@@ -43,7 +42,10 @@ public function configureEndpoints(ContainerBuilder $container, array $config)
     {
         // register API
         $container->findDefinition(ApiController::class)
-            ->addArgument($config['endpoint']['api']['auth_token'] ?? null);
+            ->addArgument($config['endpoint']['api']['exit_code']['warning'] ?? 503)
+            ->addArgument($config['endpoint']['api']['exit_code']['critical'] ?? 503)
```

Comment:
@tuxes3 `http_status_code`, not `exit_code` according to the Configuration class

Reply:
👀  I added a test for this case as well